src/NuGet.Indexing/NuGetAcronymExpansionProvider.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.Indexing
+{
+    public class NuGetAcronymExpansionProvider : IAcronymExpansionProvider
+    {
+        public static readonly IAcronymExpansionProvider Instance = new NuGetAcronymExpansionProvider();
+
+        private static readonly Dictionary<string, string[]> Acronyms = new Dictionary<string, string[]>(StringComparer.OrdinalIgnoreCase);
+
+        static NuGetAcronymExpansionProvider()
+        {
+            Acronyms.Add("xaml",  new[] { "xaml" });
```

Comment:
perhaps not for this PR, but I would have this list somewhere where it can be easily modified, without the need to redeploy the service (like a json in a blob).